config/environments/production.rb

```diff
@@ -57,7 +57,11 @@
   # config.cache_store = :mem_cache_store
 
   # Enable serving of images, stylesheets, and JavaScripts from an asset server.
-  config.action_controller.asset_host = ENV['GOVUK_ASSET_HOST']
+  config.action_controller.asset_host = Proc.new do |source, request = nil, *_|
```

Comment:
The source I link to in the commit comment explains that the second argument is apparently sometimes omitted:
```
request' is set to nil to accomodate the cases where asset_host is called in asset files (CSS and JS if you are using the asset helper tags).
```

I don't believe that we use asset helper tags - I successfully I precompiled assets in development (having first copied this code into `development.rb` so that it was executed).